src/js/module/dashboard/ListIssuesWAQ.js

```diff
@@ -79,11 +98,28 @@ class ListIssuesWAQ extends React.Component {
             }
         });
 
-        const issueCount = this.props.issues && _.size(this.props.issues);
+        // eslint-disable-next-line max-len
+        const issueCount = _.size(issuesYoungerThanOneWeek) + _.size(issuesOneWeekOld) + _.size(issuesTwoWeeksOld) + _.size(issuesThreeWeeksOld) + _.size(issuesFourWeeksOld) + _.size(issuesOlderThanFourWeeks);
```

Comment:
Couldn't think of a better way to do this, lol

Reply:
The best way to do this would probably be to use add `issueCount++` just above the switch statement and after the early return. Mind making that change?